deploy/workloads/lnic-multi-core-mica/config_lnic_mica_runtime.ini

```diff
@@ -0,0 +1,89 @@
+# RUNTIME configuration for the FireSim Simulation Manager
+# See docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst for
+# documentation of all of these params.
+
+[runfarm]
+runfarmtag=evalrunfarm
+
+f1_16xlarges=0
+m4_16xlarges=0
+f1_4xlarges=0
+f1_2xlarges=1
+
+runinstancemarket=ondemand
+spotinterruptionbehavior=terminate
+spotmaxprice=ondemand
+
+[targetconfig]
+#Set topology=no_net_config to run without a network simulation
+topology=example_1config
+no_net_num_nodes=0
+linklatency=140
+switchinglatency=0
+netbandwidth=200
+profileinterval=-1
+high_priority_obuf_size=92480
+low_priority_obuf_size=92480
+#timeout_cycles=2240
+timeout_cycles=4480
+rtt_pkts=2
+
+# This references a section from config_hwconfigs.ini
+# In homogeneous configurations, use this to set the hardware config deployed
+# for all simulators
+defaulthwconfig=firesim-rocket-quadcore-lnic-llc4mb-ddr3-90mhz
+
+[load_gen]
+use_load_gen=yes
+test_type=ONE_CONTEXT_FOUR_CORES
+service_dist_type=FIXED
```

Comment:
The service_dist_type is the key distribution for the MICA eval right?
Does FIXED mean a single key was used or was it a uniform key distribution?

Reply:
Before I was using `service_time` as the key. However, I found that too hacky. Especially because in later versions of `switch.cc` you use `service_time` to calculate timeouts, and I don't want to interfere with that.

I added a new param `service_key`. The distribution for this is uniform from `min_service_key` to `max_service_key`.

Reply:
Oh yes, glad you caught that.

Reply:
However, it requires some changes to `switch.cc`:
```
529 uint64_t get_service_key() {
530   return (*service_key_uniform_dist)(*dist_rand);
531 }

548 void send_load_packet(uint16_t dst_context, uint64_t service_time, uint64_t sent_time, uint64_t service_key) {
```